Moonscraper Chart Editor/Assets/Scripts/Game/Controllers/TimelineMovementController.cs

```diff
@@ -132,7 +132,7 @@ void Update()
                 float sign = Mathf.Sign(middleClickDragPercentageDelta.y);
 
                 Texture2D cursorTexture = sign >= 0 ? dragCursorPositive : dragCursorNegative;
-                Cursor.SetCursor(cursorTexture, Vector2.zero, CursorMode.ForceSoftware);
+                Cursor.SetCursor(cursorTexture, Vector2.zero, CursorMode.Auto);
```

Comment:
Upon testing this in-editor on Windows there is a side-effect of the up/down cursor image size being much bigger than expected. See screenshots below. 

ForceSoftware (original)-
![image](https://github.com/FireFox2000000/Moonscraper-Chart-Editor/assets/25739756/6a0ae5db-baa0-4c21-8a73-5d515126d393)

Auto (with change)-
![image](https://github.com/FireFox2000000/Moonscraper-Chart-Editor/assets/25739756/fe82399c-0e68-426d-9bd4-53e596086227)

Cursor images found at "\Assets\Art\UI\drag_cursor_negative.png" and "\Assets\Art\UI\drag_cursor_positive.png"

Reply:
Latest commit should fix that on Windows. The platform is determined on start and changes the mode to Auto when the Linux player is detected. In the editor, the bug doesn't exist so there is no need to check there.